Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,247 @@
+Feature: Test the validations done on fields from PlatformUI - text line fieldtype
+    In order to validate the text line fieldtype
+    As an Editor user
+    I need to be able to create and update content with text line fieldtypes
+
+    Background:
+       Given I am logged in as an 'Administrator' in PlatformUI
+
+    ##
+    # Validate the existence of expected fields from a field type when creating a content
+    ##
+    @javascript @common
+    Scenario: A Content item of an Content Type that has a Text Line Field Definition must have a Text Line Field
+        Given a Content Type with an "text line" Field exists
+        When I create a content of this type
+        Then I should see a "text line" field
+
+    @javascript @common
+    Scenario: When editing Content item, the label of a Text Line Field must be the corresponding Field Definition name
+        Given a Content Type with an "text line" with field definition name "Test text" exists
+        When I create a content of this type
+        Then I should see a "Test text" label related with the "text line" field
+
+    @javascript @common
+    Scenario: The label of a required text line field of a Content must be marked as required
+        Given a Content Type with a required "text line" with field definition name "Required" exists
+        When I create a content of this type
+        Then the "Required" field should be marked as required
+
+    ##
+    # Creating Content using a Content Type that has a text line Field Type
+    ##
+    @javascript @common
+    Scenario: A Content item with a Text Line Field can be published
+        Given a Content Type with an "text line" Field exists
+        When I create a content of this type
+        And I set "Test text" as the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript @common
+    Scenario: Creating a text line Field with an empty value works
+        Given a Content Type with an "text line" Field exists
+        When I create a content of this type
+        And I set an empty value as the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript @edge
+    Scenario: Creating a required text line Field fails validation when using an empty value
+        Given a Content Type with a required "text line" Field exists
+        When I create a content of this type
+        And I set an empty value as the Field Value
+        And I publish the content
+        Then Publishing fails with validation error message "This field is required"
+
+    @javascript @common
```

Comment:
This one, and the one below, should be phrased to cover the part specific to PlatformUI (client side validation).